### PR TITLE
S1t retrieve

### PR DESCRIPTION
**Pull request by SteeleSR**

### CODE REVIEW

File: dbconn.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+/**
+ * Function which creates a connection to the database
+ *
+ * @return PDO returns painting database
+ */
+
+function getDb():PDO
+{
+    $db = new PDO('mysql:host=db; dbname=sam-collection', 'root', 'password');
```
Comment (khorh) [inline dbconn.php:11]: I think you have changed your database file name to sam-collection-revised

File: index.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
+<?php
+
+require_once "dbconn.php";
```
Comment (khorh) [inline index.php:3]: This file name is missing from .gitignore

File: dbconn.php (hunk 1 above)
Comment (SteeleSR) [reply dbconn.php:11]: good spot thanks

**Review by khorh — COMMENTED** [review]
I also noticed your database could have ENUM for the fields 'place' and 'city'.  If you have time to change it!

**Pull request merged**